stages/org.osbuild.grub2

```diff
@@ -228,6 +258,8 @@ function load_video {
 }
 ${features}
 blscfg
+
+${serial}${terminal_input}${terminal_output}
```

Comment:
I think that probably should be reversed, i.e. the `blscfg` should be the last command, since that will trigger the menu.

Reply:
Maybe it can go directly next to ${features}?

Reply:
Done!